### ISSUE TITLE
Republish CheerpJ versions to capture new Albanian translations

### ISSUE THREAD

**Issue by oliver-phet** [phetsims/decaf#128]
The following translations were received Jan 2021 and the translator has requested we publish CheerpJ versions:

discharge-lamps-strings_sq.properties
efield-strings_sq.properties
electric-hockey-strings_sq.properties
forces-1d-strings_sq.properties
fourier-strings_sq.properties
ladybug-motion-2d-strings_sq.properties
maze-game-strings_sq.properties
microwaves-strings_sq.properties
motion-2d-strings_sq.properties
motion-series-strings_sq.properties
moving-man-strings_sq.properties
mri-strings_sq.properties
optical-quantum-control-strings_sq.properties
quantum-tunneling-strings_sq.properties
quantum-wave-interference-strings_sq.properties
radio-waves-strings_sq.properties
reactions-and-rates-strings_sq.properties
rotation-strings_sq.properties
semiconductor-strings_sq.properties
signal-circuit-strings_sq.properties
sound-strings_sq.properties
sugar-and-salt-solutions-strings_sq.properties
the-ramp-strings_sq.properties

@samreid assigning you to this issue, though this is not a high-priority re-deploy since the .jar files are available on the website.

**Comment by oliver-phet** [comment]
Here was the issue with information about the last redeploy we did to capture new translations: https://github.com/phetsims/decaf/issues/124

**Comment by samreid** [comment]
Here are the changes since last deploy:

```
--- Merging r75191 through r75218 into '.':
A    simulations-java/simulations/the-ramp/data/the-ramp/localization/the-ramp-strings_sq.properties
A    simulations-java/simulations/sugar-and-salt-solutions/data/sugar-and-salt-solutions/localization/sugar-and-salt-solutions-strings_sq.properties
A    simulations-java/simulations/sound/data/sound/localization/sound-strings_sq.properties
A    simulations-java/simulations/signal-circuit/data/signal-circuit/localization/signal-circuit-strings_sq.properties
A    simulations-java/simulations/semiconductor/data/semiconductor/localization/semiconductor-strings_sq.properties
A    simulations-java/simulations/rotation/data/rotation/localization/rotation-strings_sq.properties
A    simulations-java/simulations/reactions-and-rates/data/reactions-and-rates/localization/reactions-and-rates-strings_sq.properties
A    simulations-java/simulations/radio-waves/data/radio-waves/localization/radio-waves-strings_sq.properties
A    simulations-java/simulations/quantum-wave-interference/data/quantum-wave-interference/localization/quantum-wave-interference-strings_sq.properties
A    simulations-java/simulations/quantum-tunneling/data/quantum-tunneling/localization/quantum-tunneling-strings_sq.properties
A    simulations-java/simulations/optical-quantum-control/data/optical-quantum-control/localization/optical-quantum-control-strings_sq.properties
A    simulations-java/simulations/mri/data/mri/localization/mri-strings_sq.properties
A    simulations-java/simulations/moving-man/data/moving-man/localization/moving-man-strings_sq.properties
A    simulations-java/simulations/motion-series/data/motion-series/localization/motion-series-strings_sq.properties
A    simulations-java/simulations/motion-2d/data/motion-2d/localization/motion-2d-strings_sq.properties
A    simulations-java/simulations/microwaves/data/microwaves/localization/microwaves-strings_sq.properties
A    simulations-java/simulations/maze-game/data/maze-game/localization/maze-game-strings_sq.properties
A    simulations-java/simulations/ladybug-motion-2d/data/ladybug-motion-2d/localization/ladybug-motion-2d-strings_sq.properties
A    simulations-java/simulations/fourier/data/fourier/localization/fourier-strings_sq.properties
A    simulations-java/simulations/forces-1d/data/forces-1d/localization/forces-1d-strings_sq.properties
A    simulations-java/simulations/electric-hockey/data/electric-hockey/localization/electric-hockey-strings_sq.properties
A    simulations-java/simulations/discharge-lamps/data/discharge-lamps/localization/discharge-lamps-strings_sq.properties
A    simulations-java/simulations/efield/data/efield/localization/efield-strings_sq.properties
U    simulations-java/simulations/bound-states/data/bound-states/localization/bound-states-strings_tr.properties
U    simulations-flash/simulations/geometric-optics/data/geometric-optics/localization/geometric-optics-strings_uk.xml
A    simulations-flash/simulations/my-solar-system/data/my-solar-system/localization/my-solar-system-strings_ru.xml
```

**Comment by samreid** [comment]
From that list, here are the sims and translations that should be deployed:

```
bound-states-strings: tr
discharge-lamps-strings: sq
efield-strings: sq
electric-hockey-strings: sq
forces-1d-strings: sq
fourier-strings: sq
ladybug-motion-2d-strings: sq
maze-game-strings: sq
microwaves-strings: sq
motion-2d-strings: sq
motion-series-strings: sq
moving-man-strings: sq
mri-strings: sq
optical-quantum-control-strings: sq
quantum-tunneling-strings: sq
quantum-wave-interference-strings: sq
radio-waves-strings: sq
reactions-and-rates-strings: sq
rotation-strings: sq
semiconductor-strings: sq
signal-circuit-strings: sq
sound-strings: sq
sugar-and-salt-solutions-strings: sq
the-ramp-strings: sq
```

**Comment by samreid** [comment]
I checked in the translator credits to confirm that bound-states was correctly deployed.

**Comment by samreid** [comment]
For Albanian, I confirmed the link is pointing to the JAR:

![image](https://user-images.githubusercontent.com/679486/106925744-9d667b00-66cd-11eb-98fb-99da515639af.png)

**Comment by samreid** [comment]
I deployed discharge-lamps, and confirmed that sq is a working locale, but the website doesn't yet link to cheerpj for the sq locale.  I tried logging in to see if that would bypass the cache, but it still linked to the JAR.

**Comment by samreid** [comment]
@mattpen said

> Ok, it looks like it is not correct in the database, so the translate process did not work

**Comment by mattpen** [comment]
It looks like deployDecaf.js failed to provide the "flavors" of the project in the `simulations` query parameter.  The request that was received by the website was `/services/deploy-cheerpj?project=discharge-lamps&version=1.0.0-dev.17&locales=am,ar,be,bs,da,de,el,es,es_MX,es_PE,et,eu,fa,fi,fr,gl,hr,hu,in,it,iw,ja,ka,kk,ko,ku_TR,lo,mk,mn,nb,nl,pl,pt,pt_BR,ru,sk,sq,sr,sv,th,tk,tr,uk,vi,zh_CN,zh_TW&simulations=`.  The simulations query is blank, so `flavors.join( ',' )` on line 167 of deployDecaf.js must have failed for some reason.  

@samreid - Do you still have the command line log output from the discharge-lamps deploy?  It looks like we should have logged this value on the command line (line 147).

**Comment by samreid** [comment]
> @samreid - Do you still have the command line log output from the discharge-lamps deploy?

I closed that session and didn't save the output.  I can run it again later and see what happens.

**Comment by samreid** [comment]
I was unable to get `chmod` to work under phet-admin, because it said permission denied, and I was unable to sudo as phet-admin.  So I did the chmod steps under my own account. It seemed like I was able to deploy discharge-lamps, but it still didn't appear on the webpage (even with a fake query parameter to disable cache).  I confirmed @mattpen's suspicion that the simulations list was incorrect.  I manually supplied "simulations=discharge-lamps" and it worked!  I'll look into why that was missing.
